src/R/Editor/Impl/Selection/SelectWordCommand.cs

```diff
@@ -24,60 +22,13 @@ public sealed class SelectWordCommand : ViewCommand {
         public override CommandResult Invoke(Guid group, int id, object inputArg, ref object outputArg) {
```

Comment:
So they don't automatically use `ITextStructureNavigator` in this case? Perhaps we should file a bug on VS so that they respect it in selection code.